addons/im_livechat/tests/test_get_discuss_channel.py

```diff
@@ -346,3 +346,38 @@ def test_livechat_manager_can_invite_anyone(self):
             partner_ids=self.operators[0].partner_id.ids
         )
         self.assertEqual(self_member.partner_id, self.operators[0].partner_id)
+
+    def test_livechat_operator_can_see_all_livechat_conversations(self):
+        bob_user = new_test_user(
+            self.env, "bob_user", groups="base.group_user,im_livechat.im_livechat_group_user"
+        )
+        live_chat = self.env["discuss.channel"].create(
+            {
+                "channel_type": "livechat",
+                "livechat_operator_id": self.operators[0].partner_id.id,
+                "name": "test",
+            }
+        )
+        self.assertEqual(
+            self.env["discuss.channel"].with_user(bob_user).search([("id", "=", live_chat.id)]),
+            live_chat
+        )
+
+    def test_livechat_operator_can_see_all_livechat_conversations_members(self):
+        bob_user = new_test_user(
+            self.env, "bob_user", groups="base.group_user,im_livechat.im_livechat_group_user"
+        )
+        live_chat = self.env["discuss.channel"].create(
+            {
+                "channel_type": "livechat",
+                "livechat_operator_id": self.operators[0].partner_id.id,
+                "name": "test",
+            }
+        )
```

Comment:
The setup being exactly the same and both test having only a single assert, I think it would be fine to merge them into a single test.